osgtest/library/java.py

```diff
@@ -26,22 +26,22 @@ def select_ver(java_type, version):
     """Select the specified version of java in the alternatives"""
     alternatives = _run_alternatives(java_type, '\n', 'find %s version for selection' % java_type)
     selection = ''
-    for alt in re.finditer(r'(\d+)\s+(\/.*\n)', alternatives):
-        if version in alt.group(2):
-            selection = alt.group(1)
+    for alt in alternatives.split('\n'):
+        if version in alt:
+            selection = re.match(r'[\s\*\+]*(\d+)\s+', alt).group(1)
```

Comment:
Well, historically Scientific Linux gets changes after RHEL and CentOS, so perhaps SL 7 will get this change, too, at some point. In any case, my underlying question was whether the new code works for both cases correctly.